distribution/lib/Standard/Base/0.0.0-dev/src/Widget_Helpers.enso

```diff
@@ -19,6 +19,12 @@ make_delimiter_selector : Widget
 make_delimiter_selector =
     make_single_choice [',', ';', '|', ['{tab}', "'\t'"], ['{space}', "' '"], ['{newline}', "['\n', '\r\n', '\r']"], '_', ['Custom', "'?'"]]
 
+ ## PRIVATE
+      Creates a Single_Choice Widget for file read delimiters.
+make_file_read_delimiter_selector : Widget
+make_file_read_delimiter_selector =
+        make_single_choice [',', ';', '|', ['{tab}', "'\t'"], ['{space}', "' '"], ['{none}', "''"], '_', ['Custom', "'?'"]]
```

Comment:
Hmm, given that this is used _only_ within `Delimited_Format`, I'm wondering if maybe we should move this helper function there? We sometimes do define widgets next to their usage site.

It makes sense to keep it here if we expect it may also be used somewhere else. Do we?

Reply:
Yeah I wondered the same. In fact I had it next to Delimited_Format then moved it back out. My thinking was that although these 2 are different, they are similar and if we added something to one we would want to add it to the other (or at least consider the possibility). But I don't have a strong feeling either way.

Reply:
> although these 2 are different, they are similar and if we added something to one we would want to add it to the other (or at least consider the possibility)

That is a very good point that I did not consider. I think that indeed makes it worth to keep them next to each other.